entity-framework/core/what-is-new/ef-core-5.0/whatsnew.md

```diff
@@ -2,17 +2,659 @@
+
+Finally, EF Core will still create an instance of a required dependent even when all the columns for the required dependent have null values.
+
+### Options for migration generation
+
+EF Core 5.0 introduces greater control over generation of migrations for different purposes. This includes the ability to:
+
+* Know if the migration is being generated for a script or for immediate execution
+* Know if an idempotent script is being generated
+* Know if the script should exclude transaction statements (See _Migrations scripts with transactions_ below.)
+
+This behavior is specified by an the `MigrationsSqlGenerationOptions` enum, which can now be passed to `IMigrator.GenerateScript`.
+
+Also included in this work is better generation of idempotent scripts with calls to `EXEC` on SQL Server when needed. This work also enables similar improvements to the scripts generated by other database providers, including PostgreSQL.
+
+### Migrations scripts with transactions
+
+SQL scripts generated from migrations now contain statements to begin and commit transactions as appropriate for the migration. For example, the migration script below was generated from two migrations. Notice that each migration is now applied inside a transaction.
+
+```sql
+BEGIN TRANSACTION;
+GO
+
+CREATE TABLE [Groups] (
+    [Id] int NOT NULL IDENTITY,
+    [Name] nvarchar(max) NULL,
+    CONSTRAINT [PK_Groups] PRIMARY KEY ([Id])
+);
+GO
+
+CREATE TABLE [Members] (
+    [Id] int NOT NULL IDENTITY,
+    [Name] nvarchar(max) NULL,
+    [GroupId] int NULL,
+    CONSTRAINT [PK_Members] PRIMARY KEY ([Id]),
+    CONSTRAINT [FK_Members_Groups_GroupId] FOREIGN KEY ([GroupId]) REFERENCES [Groups] ([Id]) ON DELETE NO ACTION
+);
+GO
+
+CREATE INDEX [IX_Members_GroupId] ON [Members] ([GroupId]);
+GO
+
+INSERT INTO [__EFMigrationsHistory] ([MigrationId], [ProductVersion])
+VALUES (N'20200910194835_One', N'6.0.0-alpha.1.20460.2');
+GO
+
+COMMIT;
+GO
+
+BEGIN TRANSACTION;
+GO
+
+EXEC sp_rename N'[Groups].[Name]', N'GroupName', N'COLUMN';
+GO
+
+INSERT INTO [__EFMigrationsHistory] ([MigrationId], [ProductVersion])
+VALUES (N'20200910195234_Two', N'6.0.0-alpha.1.20460.2');
+GO
+
+COMMIT;
+```
+
+As mentioned in the previous section, this use of transactions can be disabled if transactions need to be handled differently.
+
+### See pending migrations
+
+This feature was contributed from the community by [@Psypher9](https://github.com/Psypher9). Many thanks for the contribution!
+
+The `dotnet ef migrations list` command now shows which migrations have not yet been applied to the database. For example:
+
+```
+ajcvickers@avickers420u:~/AllTogetherNow/Daily$ dotnet ef migrations list
+Build started...
+Build succeeded.
+20200910201647_One
+20200910201708_Two
+20200910202050_Three (Pending)
+ajcvickers@avickers420u:~/AllTogetherNow/Daily$
+```
+
+In addition, there is now a `Get-Migrations` command for the Package Manager Console with the same functionality.
```

Comment:
```suggestion
In addition, there is now a `Get-Migration` command for the Package Manager Console with the same functionality.
```

Reply:
I just can't get this right! :facepalm:

Reply:
lol, to be fair `Get-Migrations` was the name in EF6